lib/js/atoms/SurveyQuestion/SurveyQuestionTextarea.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<textarea
+		ref="textarea"
+		class="surveyQuestionTextarea"
+		:disabled="disabled"
+		:placeholder="placeholder"
+		:value="value"
+		@input="onInput"
+	></textarea>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.surveyQuestionTextarea {
+	@include textL;
+
+	border: 1px solid $color-mischka-gray;
+	box-sizing: border-box;
+	box-shadow: inset 0 1px 3px $color-minor-supporting;
+	border-radius: $radius-s;
+	min-height: 2em;
+	padding: $space-xxs;
+	resize: none;
+
+	&:disabled {
+		background: $color-total-white;
+		color: $color-minor-supporting;
+	}
+}
+</style>
+
+<script lang="ts">
+export default {
+	name: 'SurveyQuestionTextarea',
+	props: {
+		value: {
+			type: String,
+			required: true,
+		},
+		placeholder: {
+			type: String,
+			default: 'Wpisz swoją odpowiedź',
+		},
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	watch: {
+		value() {
+			this.$refs.textarea.style.height = 'auto';
+			this.$refs.textarea.style.height = this.$refs.textarea.scrollHeight + 'px';
```

Comment:
Why do we set it twice? If it's not a mistake, let's explain it in a comment.

Reply:
Should we throttle this?

Reply:
IMO, user isn't typing fast enough to impact performance significantly, there is no need to throttle. explanation: [`a33d7c9` (#71)](https://github.com/bethinkpl/design-system/pull/71/commits/a33d7c9837d2f8f572dbd80232922951c352caea)